modules/determining-upgrade-viability-cv-conditiontype.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * updating/index.adoc
+
+:_content-type: CONCEPT
+[id="Understanding_clusterversion_conditiontypes_{context}"]
```

Comment:
This ID needs revised. eg: 

[id="understanding-cluster-version-condition-types_{context}"]

They can't have capital letters in them, and you only use underscores (_) before {context}. Capital letters can break Pantheon (which happened to me recently :D ) 

While not a rule, generally the ID of the module is a variation of name of the module. It looks like the name of the module in this case is

determining-upgrade-viability-cv-conditiontype.adoc

so you might consider updating your ID to reflect that.

Reply:
Thank you! Resolved :)